frontend/src/modules/report/FeatureListContainer/FeatureListContainer.jsx

```diff
@@ -39,211 +40,194 @@ class FeatureListContainer extends Component {
     });
   }
 
-  handleTagAssigned(restId, tag, prevUserName, userName) {
+  handleTagAssigned = (restId, tag, prevUserName, userName) => {
     var newUserName = userName;
     if (prevUserName === userName) {
       newUserName = null;
     }
     setTagAssignmentData(restId, new TagAssignmentPatch(tag, newUserName)).then(response => {
-      if (!response || response.status !== 200) {
+      if (!response || !response.ok) {
         this.setStateForTagUser(tag, prevUserName);
-        this.props.handleErrorMessageDisplay();
       }
     });
     this.setStateForTagUser(tag, newUserName);
-  }
+  };
 
-  handleWarningShow(...args) {
+  handleWarningShow = (...args) => {
     this.setState({ warningArgs: args });
-  }
+  };
 
-  handleWarningClosed() {
+  handleWarningClosed = () => {
     this.setState({ warningArgs: null });
-  }
+  };
 
-  handleTagIgnore(product, tagName) {
+  handleTagIgnore = (product, tagName) => {
     setIgnoredTag(product, { tagName: tagName }).then(response => {
-      if (!response || response.status !== 200) {
+      if (!response || !response.ok) {
         this.setIgnoreStateForTag(tagName);
-        this.props.handleErrorMessageDisplay();
       }
     });
     this.setIgnoreStateForTag(tagName);
-  }
+  };
 
   setIgnoreStateForTag(tagName) {
     this.setState(prevState => {
       const newFeatureList = prevState.featureList.clone();
       newFeatureList.toggleIgnoreForTag(tagName);
-      return Object.assign({}, prevState, {
-        featureList: newFeatureList,
-      });
+      return { ...prevState, featureList: newFeatureList };
     });
   }
 
   filterTags(userName) {
-    var newTagList = this.state.featureList.tagList;
+    const { featureList, isAssignedTagsView, selectedStatus } = this.state;
+    var newTagList = featureList.tagList;
 
-    if (this.state.isAssignedTagsView) {
+    if (isAssignedTagsView) {
       newTagList = newTagList.filter(tag => tag.userName === userName);
     }
 
     return newTagList.filter(
       tag =>
-        (this.state.selectedStatus.passed && tag.containsPassed) ||
-        (this.state.selectedStatus.failed && tag.containsFailed) ||
-        (this.state.selectedStatus.undefined && tag.containsUndefined) ||
-        (this.state.selectedStatus.skipped && tag.containsSkipped)
+        (selectedStatus.passed && tag.containsPassed) ||
+        (selectedStatus.failed && tag.containsFailed) ||
+        (selectedStatus.undefined && tag.containsUndefined) ||
+        (selectedStatus.skipped && tag.containsSkipped)
     );
   }
 
-  renderAssignedTagsSwitch(classes) {
+  renderAssignedTagsSwitch() {
+    const { classes } = this.props;
+    const { isEditMode, isAssignedTagsView } = this.state;
     return (
       <>
-        <Tooltip title={this.state.isEditMode ? 'Turn Edit Mode Off' : 'Turn Edit Mode On'} placement="top">
+        <Tooltip title={isEditMode ? 'Turn Edit Mode Off' : 'Turn Edit Mode On'} placement="top">
```

Comment:
Might be good to put these tooltips into translations at some point.

Reply:
Can do